Sources/App/Views/Plot+Extensions.swift

```diff
@@ -51,6 +51,22 @@ extension Node where Context: HTML.BodyContext {
         .element(named: "tab-bar", nodes: nodes)
     }
 
+    static func spiOverflowingList(overflowMessage: String,
+                                   overflowHeight: Int,
+                                   _ nodes: Node<HTML.ListContext>...) -> Self {
+        // Note: The `overflowHeight` is a magic number that needs some experimentation each
+        // time this tag is used. It's the exact size in pixels of the collapsed element. If
+        // incorrect, the "show more" button will show up unnecessarily as it may overflow by
+        // just a single invisible pixel.
```

Comment:
👌 

<img width="559" alt="CleanShot 2022-02-22 at 17 18 20@2x" src="https://user-images.githubusercontent.com/65520/155173804-caa62b64-6d74-4635-93e9-7200c3713d54.png">


https://github.com/SwiftPackageIndex/Pipelines/blob/295e2bdde86abdf8d95fd999a5fab0a7f05fc2eb/Sources/HomeView/SidebarView.swift#L367

Reply:
Magic numbers are fine as long as you explain them... Right? 😂